src/sounddeviceportaudio.h

```diff
@@ -96,7 +96,7 @@ class SoundDevicePortAudio : public SoundDevice {
     ControlObjectSlave* m_pMasterAudioLatencyOverload;
     int m_underflowUpdateCount;
     static volatile int m_underflowHappend;
-    int m_nsInAudioCb;
+    qint64 m_nsInAudioCb;
```

Comment:
This one wasn't a performance change -- it was for correctness on 32-bit platforms (and on Windows where int is 32-bits no matter what). It overflows for values larger than 2 seconds :).